packages/table/src/interactions/menus/copyCellsMenuItem.tsx

```diff
@@ -16,14 +16,14 @@
 
 import * as React from "react";
 
-import { MenuItem, MenuItemProps } from "@blueprintjs/core";
+import { MenuItem2, MenuItem2Props } from "@blueprintjs/popover2";
 
 import { Clipboard } from "../../common/clipboard";
 import { TABLE_COPY_FAILED } from "../../common/errors";
 import { Regions } from "../../regions";
 import { IMenuContext } from "./menuContext";
 
-export interface ICopyCellsMenuItemProps extends MenuItemProps {
+export interface ICopyCellsMenuItemProps extends MenuItem2Props {
```

Comment:
Note that this is a small breaking change, but in practice I think it should be ok for consumers since I didn't see any usage of `popoverProps: Partial<IPopoverProps>` for this component in any of our code bases.